exercise/exercise_models.py

```diff
@@ -651,7 +652,7 @@ def delta_in_minutes_from_closing_to_date(self, future_date):
         # -> convert future_date to same format as module_close
         string_date = str(future_date)[:16]
         converted = timezone.make_aware(
-                datetime.datetime.strptime(string_date, '%Y-%m-%d %H:%M'),
+                parse_datetime(string_date),
```

Comment:
I changed this to `parse_datetime` so we wouldn't need to have this hardcoded format string here, plus the format string was not consistent with the way dates are sent from HTML forms (they use the T-format).